packages/devtools_app/lib/src/screens/memory/panes/diff/controller/snapshot_item.dart

```diff
@@ -86,9 +87,13 @@ class SnapshotDataItem extends SnapshotItem
 
   Future<void> loadHeap(HeapGraphLoader loader) async {
     assert(_heap == null);
+    SnapshotTimer.instance.maybePrint('Loading');
```

Comment:
instead of creating a new timer class, we should just record this timing information to our analytics. We could either use the ga.timeAsync methods, or we could create local variables to record the time and then send a single analytics event with the timing profile info.

Reply:
This class is designed to see numbers for a concrete local experimental run. 
Can our analytics be used for this goal? 
If yes, where can I find steps?

Reply:
If the only goal is to add a way to print timing information for local development, then we can use the existing helpers `debugTimeSync` and `debugTimeAsync`: https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/shared/development_helpers.dart/#L267-L297

Reply:
These two classes give gust one number and require separate function.
I do not want to do refactoring, I want to be able to put checkpoints in any places.
I moved the class DebugTimer to be at the same place as these two, and added documentation.